packages/amplify-provider-awscloudformation/src/initializer.ts

```diff
@@ -107,9 +107,10 @@ export const run = async (context: $TSContext): Promise<void> => {
             external: true,
           },
         });
-        sandboxNode.run(overrideCode).override(configuration);
+        await sandboxNode.run(overrideCode).override(configuration);
       }
     } catch (e) {
+      // TODO: this always throws, and likely needs re-written
```

Comment:
a note on this... this seems to always throw even if there aren't any issues with the files; any context on why would be useful...

Reply:
This is a sunny day scenario https://docs.amplify.aws/cli/project/override/ for this code. I wonder if we have happy path test case at all.

Reply:
So the above code doesn't apply override at all ?

Reply:
This code does not throw if overriding the root stack and `override.js` exists and can be read and compile.

The `try` fails typically because `override.js` does not exist.